src/enforcer.js

```diff
@@ -26,7 +26,43 @@ const {
 
 const {slice} = String.prototype;
 
-const UrlConstructor = URL.prototype.constructor;
+// No URL in IE 11.
+const UrlConstructor = typeof window.URL == 'function' ?
+    URL.prototype.constructor :
+    null;
+
+const stringifyForRangeHack = (function(doc) {
+  const r = doc.createRange();
+  // In IE 11 Range.createContextualFragment doesn't stringify its argument.
+  const f = r.createContextualFragment(/** @type {string} */ (
+      {toString: () => '<div></div>'}));
+  return f.childNodes.length == 0;
+})(document);
+
+/**
+ * Return object constuctor name
+ * (their function.name is not available in IE 11).
+ * @param {*} fn
+ * @return {string}
+ * @private
+ */
+const getConstructorName_ = document.createElement('div').constructor.name ?
+    (fn) => fn.name :
+    (fn) => ('' + fn).match(/^\[object (\S+)\]$/)[1];
+
+// Hijacking window.open on IE 11 is unreliable.
```

Comment:
It simply seems not to work, I tried hooking into various prototype / constructor combinations. This might be also breaking in tests only (perhaps it breaks under some instrumentation), but I decided to leave it out for now and revisit, hence the TODO.